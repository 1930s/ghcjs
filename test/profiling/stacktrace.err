--- a
Stack trace:
  Main.errorM.\ (stacktrace.hs:13:22-54)
  Main.errorM (stacktrace.hs:13:1-54)
  Main.foo.\.\ (stacktrace.hs:27:23-30)
  Main.foo.\ (stacktrace.hs:27:16-33)
  Main.foo (stacktrace.hs:27:1-36)
  Main.bar (stacktrace.hs:24:1-20)
  Main.runM.(...) (stacktrace.hs:16:20-31)
  Main.runM (stacktrace.hs:16:1-36)
  Main.main (stacktrace.hs:(29,1)-(37,36))
  Main.CAF (<entire-module>)
