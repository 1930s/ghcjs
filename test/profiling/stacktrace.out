["Main.CAF (<entire-module>)","Main.main (stacktrace.hs:(29,1)-(37,36))"]
[]
[]
["MAIN.DONT_CARE (<built-in>)"]
["Main.CAF (<entire-module>)","Main.main (stacktrace.hs:(29,1)-(37,36))"]
["Main.CAF (<entire-module>)","Main.main (stacktrace.hs:(29,1)-(37,36))","Main.someF (stacktrace.hs:19:1-35)","Main.replicateM (stacktrace.hs:22:1-41)"]
["MAIN.DONT_CARE (<built-in>)"]
